README.md

```diff
@@ -1,193 +1,198 @@
-  - [Explore metrics to traces to logs](#explore-metrics-to-traces-to-logs)
-    - [Explore LogQL V2](#explore-logql-v2)
-  - [Disable TNS cluster](#disable-tns-cluster)
-  - [Remove TNS cluster](#remove-tns-cluster)
-  - [Troubleshooting](#troubleshooting)
-  - [Contributing guidelines](#contributing-guidelines)
-    - [Modify TNS application:](#modify-tns-application)
-    - [Update Grafana dashboards and kubernetes infrastructure:](#update-grafana-dashboards-and-kubernetes-infrastructure)
-
 ## Overview
 
-The New Stack (TNS) is a simple three-tier demo application, fully instrumented with the 3 pillars of observability: metrics, logs, and traces. It offers an insight on what a modern observability stack looks like and experience what it's like to pivot among different types of observability data.
+The New Stack (TNS) is a simple demo application, fully instrumented with the 3 pillars of observability: metrics, logs, and traces. It offers an insight into what a modern observability stack looks like, and what it's like to pivot among different types of observability data.
 
 ![TNS workflow](./img/TNS-Arc-Diagram-1.png)
 
 The TNS app is an example three-tier web app built by Weaveworks. It consists of a data layer, application logic layer, and load-balancing layer. To learn more about it, see [How To Detect, Map and Monitor Docker Containers with Weave Scope from Weaveworks](https://thenewstack.io/how-to-detect-map-and-monitor-docker-containers-with-weave-scope-from-weaveworks/).
 
 The instrumentation for the TNS app is as follows:
 
-- Metrics: Each tier of the TNS app exposes metrics on /metrics endpoints, which are scraped by the Grafana Agent. Additionally, these metrics are additionally tagged with exemplar information. The Grafana Agent then writes these metrics to a Prometheus (with remote-read enabled) for storage. [While the Prometheus could scrape metrics from the TNS App directly, the demo is configured to make the Agent the central point through which metrics, logs, and traces are collected. The Prometheus can be substituted for any backend which accepts Prometheus remote write, such as Thanos or Cortex.]
-- Logs: Each tier of the TNS app writes logs to standard output or standard error which is captured by Kubernetes, which are then collected by the Grafana Agent, which forwards them on to Loki for storage. 
+- Metrics: Each tier of the TNS app exposes metrics, tagged with [exemplar](https://grafana.com/docs/grafana/latest/basics/exemplars/) information, on `/metrics` endpoints. These are scraped by the Grafana Agent and then written to a Prometheus (with remote-read enabled) for storage. (While the Prometheus could scrape metrics from the TNS App directly, the demo is configured to make the Agent the central point through which metrics, logs, and traces are collected. The Prometheus can be substituted for any backend which accepts Prometheus remote write, such as Thanos or Cortex.)
```

Comment:
```suggestion
- Metrics: Each tier of the TNS app exposes metrics, tagged with [exemplar](https://grafana.com/docs/grafana/latest/basics/exemplars/) information, on `/metrics` endpoints. These are scraped by the Grafana Agent and then written to a Prometheus (with remote-read enabled) for storage. (While the Prometheus could scrape metrics from the TNS App directly, it is configured to collect metrics, logs, and traces through the Agent. You can substitute Prometheus with any backend which accepts Prometheus remote write, such as Thanos or Cortex.)
```

Reply:
The correction makes it seem the Prometheus is configured to collect metrics through the Agent, but it is the agent that is configured to collect the metrics and send them to Prometheus.

```suggestion
- Metrics: Each tier of the TNS application exposes metrics, tagged with [exemplar](https://grafana.com/docs/grafana/latest/basics/exemplars/) information, on `/metrics` endpoints. These are scraped by the Grafana Agent and then written to a Prometheus (with remote-read enabled) for storage. While the Prometheus could scrape metrics from the TNS application directly, the Agent is configured to be the central point through which metrics, logs, and traces are collected. You can replace the Prometheus with any backend which accepts Prometheus remote write, such as Thanos, Cortex, or Mimir.
```